price-feeder/oracle/provider/kraken.go

```diff
@@ -91,7 +93,7 @@ func (p KrakenProvider) GetTickerPrices(tickers ...string) (map[string]sdk.Dec,
 		)
 	}
 
-	tickerPrices := make(map[string]sdk.Dec, len(tickers))
+	tickerPrices := make(map[string]TickerPrice, len(tickers))
 	for _, t := range tickers {
 		// TODO: We may need to transform 't' prior to looking it up in the response
```

Comment:
Not sure I understand this todo - do we still need to transform this?

Reply:
The TODO is because Kraken might not necessarily treat ATOM as ATOM. E.g. they treat BTC as XBTC: https://api.kraken.com/0/public/Ticker?pair=XBTUSD

So in the case of Bitcoin, the value provided to this function would be `BTCUSD`, however, the response contains `XXBTZUSD`, the lookup would fail! Hence, we'd need to hardcode a transformation from XXBTZUSD to BTCUSD and thus the TODO I added. The reason we haven't done anything yet, is because BTC is the only ticker I know of for Kraken that they do this for and we won't support BTC.